hadoop-hdds/server-scm/src/main/java/org/apache/hadoop/hdds/scm/server/SCMCertStore.java

```diff
@@ -142,7 +144,7 @@ public Optional<Long> revokeCertificates(
       CRLReason reason,
       Date revocationTime,
       CRLApprover crlApprover)
-      throws IOException {
+      throws IOException, TimeoutException {
```

Comment:
The interface for `revokeCertificates` needs to be modified if used.  The `CRLApprover `object should not be passed in the ratis transaction, instead what is processed should be redesigned to be pre-signed by the CA.

Reply:
@neils-dev thanks for taking a look at it. These changes are covered as part of the follow-up jiras.

Reply:
@neils-dev thanks for taking a look at it. These changes are covered as part of the follow-up jiras.